### ISSUE TITLE
question about evaluation 

### ISSUE THREAD

**Issue by yuyijie1995** [WeitaoVan/L-GM-loss#15]
In your tensorflow code,I found you use the dist as the logits_eval and take the argmax as the right label . But the dist is the distance between the example feature  and the distribution mean ,why the biggest distance distribution is the right label?
